Detectors/FIT/simulation/include/FITSimulation/Digitizer.h

```diff
@@ -35,28 +35,33 @@ class Digitizer
   void setEventTime(double value) { mEventTime = value; }
   void setEventID(Int_t id) { mEventID = id; }
   void setSrcID(Int_t id) { mSrcID = id; }
-  Int_t getCurrentTimeFrame() const { return mTimeFrameCurrent; }
-  void setCurrentTimeFrame(Double_t value) { mTimeFrameCurrent = value; }
+  void setBC(Int_t bc) { mBC = bc; }
+  void setOrbit(Int_t orbit) { mOrbit = orbit; }
+
+  void setTriggers(Digit* digit);
+  void smearCFDtime(Digit* digit);
```

Comment:
do you modify the digit in this function? If not, consider passing digit as `const& `. Applies also to `setTriggers.`